stats/kafka/config.go

```diff
@@ -0,0 +1,127 @@
+/*
+ *
+ * k6 - a next-generation load testing tool
+ * Copyright (C) 2016 Load Impact
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package kafka
+
+import (
+	"encoding/json"
+
+	"github.com/kubernetes/helm/pkg/strvals"
+	"github.com/pkg/errors"
+)
+
+type ConfigFields struct {
+	// Connection.
+	Brokers []string `json:"brokers" envconfig:"KAFKA_BROKERS"`
+
+	// Samples.
+	Topic  string `json:"topic" envconfig:"KAFKA_TOPIC"`
```

Comment:
Can you switch to null-able fields for the config fields, something like [this](https://github.com/loadimpact/k6/blob/cloud-aggregation/stats/cloud/config.go#L32-L36) or [this](https://github.com/loadimpact/k6/blob/master/lib/options.go#L205-L206)? I know that the current config for the influxdb collector is not that way, but that's something that [we'll soon fix](https://github.com/loadimpact/k6/issues/586).